aruco_ros/launch/marker_pub.launch

```diff
@@ -0,0 +1,18 @@
+<launch>
+
+    <arg name="markerSize"      default="0.05"/>    <!-- in m -->
+    <arg name="side"             default="left"/>
+    <arg name="ref_frame"       default="base"/>  <!-- leave empty and the pose will be published wrt param parent_name -->
+
+
+    <node pkg="aruco_ros" type="marker_pub" name="aruco_marker_publisher">
```

Comment:
The type here should be the name of the binary that you compile the node into: `marker_publisher`
The launch file as of now fails